src/RequestPage.php

```diff
@@ -323,9 +323,11 @@ function requestHistoryDisplay(AccountRequest &$accountRequest, array &$history,
 			Html::element('span', [], wfMessage(actions[$historyEntry->action]['message'])->text())
 		]);
 
-		$row .= Html::closeElement('h5');
-		$row .= Html::element('p', [], $historyEntry->comment);
-		$row .= Html::closeElement('div');
+		if ($historyEntry->comment != null) {
```

Comment:
A lot closer, but still a few issues:
1. This should be checking `$historyEntry->performer` and not `$historyEntry->comment` (think about what the actual data is: the "comment" property is the comment, and if the comment is null then we will have big problems beyond showing links or whatever)
2. This still needs to show the comment normally if it was made by the requester (i.e. `$historyEntry->performer == null`), while this just won't show it at all.
3. Don't call `htmlspecialchars` at all, as that will escape everything *before* passing it into the parser, which already escapes it for you as part of the parsing. This would escape everything twice, so like if your comment is "1 < 2", the display would show "1 &lt; 2" or something like that..